src/bin/falcon-config.sh

```diff
@@ -68,18 +68,6 @@ case $type in
     for i in `ls ${BASEDIR}/server/webapp`; do
       FALCONCPPATH="${FALCONCPPATH}:${i}/WEB-INF/lib/*"
     done
-    # In future we should refactor the CP directories between server and client
-    HADOOPDIR=`which hadoop`
-    if [ "$HADOOP_HOME" != "" ]; then
-      echo "Hadoop home is set, adding libraries from '${HADOOP_HOME}/bin/hadoop classpath' into falcon classpath"
-      FALCONCPPATH="${FALCONCPPATH}:`${HADOOP_HOME}/bin/hadoop classpath`"
```

Comment:
Should the update of FALCONCPPATH be kept here? It is used in later part of the code. Same for the one in "elif".